src/components/app.pcss

```diff
@@ -4,7 +4,7 @@
 }
 
 .app {
-  background-color: #f2f2f2;
+  background-color: var(--light-grey);
```

Comment:
Still pretty specific (...grey) - maybe something like primary/secondary?
If someone whats to style it differently later

Reply:
Using primary-color and primary-color-dark!